operator/k8s_identity.go

```diff
@@ -84,6 +85,9 @@ func identityGCIteration() {
 				"nodes":            identity.Status.Nodes,
 			}).Debug("Deleting unused identity")
 			deleteIdentity(identity)
+		} else {
+			// If the identity is alive then mark it as alive
+			identityHeartbeat.MarkAlive(identity.Name, timeNow)
```

Comment:
This breaks the expiration logic. Given that the identity GC interval is smaller than the identity heartbeat duration, identities will be continuously marked alive and never be expired.

Reply:
Discussed offline, fixed.